templates/configuration/graphdb-master-configmap.yaml

```diff
@@ -12,8 +12,8 @@ data:
     -Denable-context-index=true
     -Dentity-pool-implementation=transactional
     -Dhealth.max.query.time.seconds=60
-    -Dgraphdb.vhosts={{ $.Values.deployment.protocol }}://{{ if eq ($master_index | int64) 1 }}{{ $.Values.deployment.host }}{{ $.Values.graphdb.workbench.subpath }}{{ else }}{{ $.Values.deployment.host }}{{ $.Values.graphdb.workbench.subpath }}-master-{{ $master_index }}{{ end }}
-    -Dgraphdb.append.request.id.headers=true
+    -Dgraphdb.vhosts={{ $.Values.deployment.protocol }}://{{ if eq ($master_index | int64) 1 }}{{ $.Values.deployment.host }}{{ $.Values.graphdb.workbench.subpath }}{{ else }}{{ $.Values.deployment.host }}{{ $.Values.graphdb.workbench.subpath }}-master-{{ $master_index }}/{{ end }},http://graphdb-master-{{ $master_index }}:7200/
+    -Dgraphdb.external-url={{ $.Values.deployment.protocol }}://{{ if eq ($master_index | int64) 1 }}{{ $.Values.deployment.host }}{{ $.Values.graphdb.workbench.subpath }}{{ else }}{{ $.Values.deployment.host }}{{ $.Values.graphdb.workbench.subpath }}-master-{{ $master_index }}/{{ end }}    -Dgraphdb.append.request.id.headers=true
```

Comment:
a helper for this would be nice

Reply:
created https://ontotext.atlassian.net/browse/GDB-5899